larpandoracontent/LArTrackShowerId/TrackShowerIdFeatureTool.cc

```diff
@@ -504,11 +579,25 @@ void ThreeDOpeningAngleFeatureTool::Divide3DCaloHitList(const Algorithm *const p
         CaloHitList orderedCaloHitList(threeDCaloHitVector.begin(),threeDCaloHitVector.end());
         const unsigned int nhits(orderedCaloHitList.size());
 
+	//This is the original code; it divides hits into first 50% and last 50%
+	/*
         for (const CaloHit *const pCaloHit : orderedCaloHitList)
         {
             CartesianPointVector &targetVector((pointVectorStart.size() < (nhits / 2)) ? pointVectorStart : pointVectorEnd);
             targetVector.push_back(pCaloHit->GetPositionVector());
         }
+	*/
```

Comment:
Not my code block. This used to be Nick's code. Unsure as to what I should do with these. Should i just delete his code block?

Reply:
Just to add a bit more to this. Nick made this change because the original code was hard coded to take the first 50% and last 50%. Nick made the change so that the first x% and last x% was able to be defined by changing the relevant block in the xml settings file. I don't know whether this change should be pushed to the main branch.